get_id.js

```diff
@@ -38,34 +48,55 @@ if (id_type_material === 1) {
     id_type_of_filler = resp.id_type_of_filler
     if (id_type_of_filler === 1) {
         console.log("Disperse filler")
-        parse_disperse_filler(resp)
+        parse_disperse_filler(resp, filler_dispers)
     } else if (id_type_of_filler === 2) {
         console.log("Fibrous filler")
+        parse_fibrous_filler (resp, filler_fibrous)
     } else if (id_type_of_filler === 3) {
         console.log("Other filler")
+        parse_other_filler(resp, filler_other)
     }
 } else if (id_type_material === 5) {
     console.log("Thermoplastic composite materials")
 }
 
 
-let filler_dispers = ["id_type_of_filler", "marka", "manufacturer", "chemical_nature", "chemical_nature_iso", "purity_of_disperse_filler", "purity_of_disperse_filler_iso", "dispersity", "dispersity_iso", "moisture_content", "moisture_content_iso", "density", "density_iso"]
 
+function fill_form(name, placeholder) {
+    let selector = $(`input[name=${name}]:visible`)
+    selector[0].placeholder = placeholder
+}
 
-filler_dispers.forEach(function(item, i) {
-    let placeholder = resp[item]
-    console.log(placeholder)
-    fill_filler_dispers(item, placeholder)
-});
+function parse_disperse_filler (resp, filler_dispers) {
+    filler_dispers.forEach(function(item, i) {
+        let placeholder = resp[item]

```

Comment:
I think than there is not necessary let & console. 
Log may be writing into function fill_form(item,resp[item])

Reply:
+